helpers/useRedirectUnauthenticated.ts

```diff
@@ -0,0 +1,20 @@
+import { useRouter } from 'next/router'
+import { LOGIN_PATH } from '../constants'
+
+/**
+ * Redirects to login page if user is not authenticated.
+ * With next query param to redirect back to this page after login.
+ * @param {boolean} shouldRedirect Condition for redirection
+ */
+const useRedirectUnauthenticated = (shouldRedirect: boolean) => {
```

Comment:
Lets change this name, since its not being used a hook anymore

Reply:
Why is that? It doesn't make sense because the whole purpose for prefixing a hook name with `use*` is to indicate whatever this variable is, in this case, it's a react hook. 

If a function **hooks** into react lifecycle methods, then the name should be starting with the `use` prefix, by convention.